### PR TITLE
feat(client): enable itx functionality without preview feature flag

### PR DESCRIPTION
**Pull request by aqrln**
Ref: https://github.com/prisma/prisma-engines/pull/3390
Closes: https://github.com/prisma/client-planning/issues/96

### CODE REVIEW

**Comment by aqrln** [comment]
Waiting for integration binaries to be released

File: packages/client/tests/functional/_example/_matrix.ts [hunk 1]
```diff
@@ -51,25 +51,25 @@ export default defineMatrix(() => [
...
       'providerFeatures': '',
-      'previewFeatures': '"referentialIntegrity"',
+      'previewFeatures': '"relationMode"',
     },
     {
       'provider': 'mongodb',
       'id': 'String @id @default(auto()) @map("_id") @db.ObjectId',
       'providerFeatures': '"mongoDb", ',
-      'previewFeatures': '"referentialIntegrity"',
+      'previewFeatures': '"relationMode"',
```
[the remark this answers is not in the document]
Comment (janpio) [reply packages/client/tests/functional/_example/_matrix.ts:72]: The preview feature is still called `referentialIntegrity`, only its property changed to `relationMode`

[the remark this answers is not in the document]
Comment (Jolg42) [reply packages/client/tests/functional/_example/_matrix.ts:72]: Indeed, you can undo the preview feature flag rename here

[the remark this answers is not in the document]
Comment (aqrln) [reply packages/client/tests/functional/_example/_matrix.ts:72]: ah, I see, thanks, I'll revert these changes

**Review by danstarns — APPROVED** [review]

File: packages/engines/package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
   "author": "Tim Suchanek <suchanek@prisma.io>",
   "devDependencies": {
     "@prisma/debug": "workspace:*",
-    "@prisma/engines-version": "4.7.0-33.b26f09cb14ff23dc4bfd076ac1044a7976923070",
+    "@prisma/engines-version": "4.7.0-37.integration-itx-remove-preview-feature-d49c60b9b24079055b2e8e17c3a449d4424c42f0",
```
Comment (aqrln) [inline packages/engines/package.json:11]: Should be reverted before merging

**Comment by aqrln** [comment]
Engines merged in https://github.com/prisma/prisma/pull/16437, rebasing

**Pull request merged**